src/components/dialogs/sensi-parameters-selector.js

```diff
@@ -5,7 +5,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
-import React, { useState } from 'react';
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
```

Comment:
File too big to make a good review
Perhaps split it in a directory (components, hooks, ....)